questions/37374-medium-simple-odd-and-even-type-challenge/README.md

```diff
@@ -0,0 +1,2 @@
+Create OddNumbers type that generates odd numbers from 1 until 1999
+Create EvenNumbers type that generates even numbers from 0 until 1998
```

Comment:
typo, should be
Create OddNumbers type that generates odd numbers from 1 until 1999
Create EvenNumbers type that generates even numbers from 0 until 1998